e2e/wdio/helper/assertion-helper.ts

```diff
@@ -99,3 +99,4 @@ export function checkElementTextValue(element: string, expectation): void {
 export function checkTextValueContain(oldValue: string, newValue: string): void {
```

Comment:
I know it is not introduced by you but let's rename oldValue and newValue with string  and subString
Looks like these param names were inspired by specific test logic.